frontend/src/app/modules/global_search/global-search-work-packages.component.ts

```diff
@@ -145,8 +130,9 @@ export class GlobalSearchWorkPackagesComponent implements OnInit, OnDestroy, Aft
 
   public onFiltersChanged(filters:WorkPackageTableFilters) {
     if (filters.isComplete()) {
-      this.query.filters = filters.current;
-      this.queryProps = this.UrlParamsHelper.buildV3GetQueryFromQueryResource(this.query);
+      const query = this.isolatedQueryDirective.runInSpace((i, querySpace) => cloneHalResource(querySpace.query.value!)) as QueryResource;
```

Comment:
I personally would prefer to rename the parameter `i` to `injector`, as it saves the reader the question what is meant with `i`. In old school programming the `i` often referred to an iterator index and this is misleading here.

Reply:
That has since been removed in a later commit since I didn't like the `runInSpace` version at all.